mediator/src/main/kotlin/pl/edu/prz/kod/mediator/application/Application.kt

```diff
@@ -55,9 +63,12 @@ fun main() {
                         get(),
                         get(),
                         get(),
+                        get(),
                         configuration.frontendHttpUrl
```

Comment:
How about injecting whole `Configuration` object into `MediatorHttpHandler`? Then manually specifying constructor parameters like this won't be needed.

Reply:
Sure, it looks better :)